### PR TITLE
Changes to step 1 template to reorganise things.

### PR DESCRIPTION
**Pull request by unknowndomain**

### CODE REVIEW

File: main/templates/join_supporter/step1.html [hunk 1]
```diff
@@ -22,14 +22,15 @@ <h2>Join as a Supporter Member</h2>
 
     <div class="alert alert-info">
         <div class="">
-
+
```
Comment (idnorton) [inline main/templates/join_supporter/step1.html:25]: Stray whitespace crept in here

**Review by idnorton — COMMENTED** [review]
Thanks for your PR Tom, input is greatly appreciated 😀

I can see your changes here but I'm not clear on why you've restructured? Being a little picky but could you remove the stray whitespace and then think a little more about your commit message please? Perhaps https://chris.beams.io/posts/git-commit/ might help with that?

Thanks, Ian.

**Comment by castaway** [comment]
Review/tested on my local install, looks great to me (tried unsuccessfully to figure out how to render the fee to 2dp and failed, tho)

**Comment by unknowndomain** [comment]
Meh

**Pull request merged**